prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -228,12 +249,63 @@ function generateAndExportLetsEncryptCert() {
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
 
-    TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
+       TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
-}
+    sleep 2
+	#encrypt the private cert
+    encryptCerts
+
+	#copy the cert to cloud
+	gsutil cp "./letsencrypt/live/${DOMAIN}/nightly-gke-tls-integration-app-client-cert.encrypted gs://kyma-prow-secrets/"
+    #copy the private key to cloud
+    gsutil cp "./letsencrypt/live/${DOMAIN}/nightly-gke-tls-integration-app-client-key.encrypted gs://kyma-prow-secrets/"
+
+    gsutil setmeta  -h "Cache-Control:public, max-age=60" gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-key.encrypted
+    gsutil setmeta  -h "Cache-Control:public, max-age=60" gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-client.encrypted
 
+
+}
+function decryptCerts(){
+   local DOMAIN="Nightly"
+  local KYMA_KEYRING="kyma-prow"
```

Comment:
The variables shouldn't be hardcoded, the should be passed to script.

Reply:
config.yaml has variables these were added for testing and are now removed